docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.4'
+
+services:
+  http:
+    build: .
+    environment:
+      NODE_ENV: development
+    ports:
+      - "${HTTP_PORT:-4001}:4001"
+    volumes:
+      # override packaged /app for dev
+      - .:/app
+      # node_modules may not be platform compatible, use the container's.
```

Comment:
...TIL: https://docs.docker.com/storage/volumes/#populate-a-volume-using-a-container

Comment is inaccurate though as we're using `node_modules` from the image rather than the container.